sagemaker-debugger/xgboost_training_report/higgs_boson_detection.ipynb

```diff
@@ -44,7 +44,7 @@
    "metadata": {},
```

Comment:
<p>Not sure if this tool has an issue, but I don't see the profiler output displayed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2174/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>I did not displayed this on purpose since the report was pretty much blank without any issues counted. Do you think still we need to add the viz output?</p>

Reply:
<p>I think we should for parity with the XGBoost report. Maybe even consider altering the training script for this job so that issues are intentionally found?</p>